configuration/settings.xml

```diff
@@ -0,0 +1,49 @@
+<settings>
+  <pluginGroups>
+    <pluginGroup>org.jenkins-ci.tools</pluginGroup>
+    <pluginGroup>org.sonarsource.scanner.maven</pluginGroup>
+  </pluginGroups>
+
+  <localRepository>${PWD}/artifacts/m2</localRepository>
```

Comment:
Please let git ignore this directory. So others will do not push the directory again.

Reply:
Alright.